vllm/platforms/xpu.py

```diff
@@ -197,3 +197,9 @@ def supports_v1(cls, model_config: ModelConfig) -> bool:
     @classmethod
     def device_count(cls) -> int:
         return torch.xpu.device_count()
+
+    def get_global_graph_pool(self) -> Any:
+        """
+        Currently xpu does NOT support Graph model.
```

Comment:
is this just saying we don't support cudagraphs on xpu?

Reply:
yes, torch-xpu will add this feature in the future.